mods/sp/rules/aircraft.yaml

```diff
@@ -642,10 +642,11 @@ NUKECARRYALL_BOT:
 	Cargo:
 		InitialUnits: hvrtruk3_bot
 	Armament@AIaiming:
-		RequireCondition: !unloading && AImicroManage
+		RequireCondition: AImicroManage
 		Weapon: AITransportAimingDummyWeapon
 		LocalOffset: 950,0,0
 		ReloadingCondition: unloading
+		PauseOnCondition: unloading
```

Comment:
Hm. This works, but it looks quite much unintended behaviour. I would have expected armament pausing also pausing reload, but that isn't the case. YMMV.

Reply:
Emmm, I guess I found the difference between `PauseOnCondition` and `RequiresCondition` when I was doing that draging function (show change number). In TS component tower usecase about power consuming amount, if `RequiresCondition` is not satisfied, I cannot get the related power trait info class in code. However, `PauseOnCondition` can stop the trait meanwhile trait info class remains accessible in code. So I guess `PauseOnCondition` can be more safe here, if engine change the code related on reload countdown.

Reply:
In fact I was surprise when I use `RequiresCondition` and found that the trait is still ticking.